app/assets/stylesheets/forever_style_guide/modules/_pnav.scss

```diff
@@ -0,0 +1,464 @@
+
+$f-pnav__main--zindex: 1030;
+$f-pnav__side--width: 320px;
+$f-pnav__side--zindex: 1032;
+
+@mixin deals_link_base() {
+  color: $color-forever_orange;
+
+  &:hover {
+    color: $color-forever_blue;
+  }
+
+  &::after {
+    position: relative;
+    top: -5px;
+    right: -2px;
+    font-family: FontAwesome;
+    font-size: 10px;
+    content: '\f005';
+  }
+}
+
+.f-pnav {
+  width: 100%;
+  padding: 0 10px;
+  box-shadow: 0 3px 5px rgba(0, 0, 0, 0.1);
+  background-color: $color-white;
+
+  /*support store method of affixing*/
+  &.affix {
+    top: 0;
+    left: 0;
+    z-index: $f-pnav__side--zindex;
+  }
+
+  /* nested to give more specificity and override the base styleguide link styles */
+  .f-pnav__link-base {
+    text-decoration: none;
+    color: $color-gray-600;
+    @include transition_fast(color);
+
+    &.is-active,
+    &:hover {
+      color: $color-forever_blue;
+    }
+  }
+}
+
+.f-pnav__main {
+  display: flex;
+  min-height: 50px;
+  position: relative;
+  z-index: $f-pnav__main--zindex;
+}
+
+.f-pnav__logo-link {
+  display: flex;
+  align-items: center;
+}
+
+.f-pnav__logo-image {
+  height: 22px;
+  max-width: 138px;
+  @include transition_fast(height, max-width);
+}
+
+.f-pnav__ecom-list {
+  display: flex;
+  flex: 1 0 auto;
+  justify-content: center;
+  align-items: stretch;
+  margin: 0;
```

Comment:
these margin and padding shorthands are more prescriptive than is necessary, where margin-top, margin-bottom, and padding-left are what is actually needed to be reset to 0

Reply:
I suspect there are other instances of this here, so I'll let this comment suffice to cover.